### DISCUSSION TITLE
Zoomer frame information

### DISCUSSION THREAD

**Discussion by thhart** [GSI-CS-CO/chart-fx#389] [Q&A] [answered]
I would like to display the axis boundaries of the zoom frame currently visible. Can this be achieved somehow?
For instance I have a date axis for X and would like to see the left and right framing dates while zooming?
Right now I use the standard Zoomer plugin.

**Comment by wirew0rm** [comment, marked as answer]
If I understand your question correctly, you should be able to just use the Min/Max property of the xAxis. You can also add listeners to the axis (beware that they are not JavaFX listeners, so they will not run in the javafx application thread and you need to wrap UI updates in runFx() calls). In the case of date axis you probably also want to get the LabelFormatter from the axis, as dates are just utc millisecond timestamps from chart-fx's POV.
